corelib-definitions/src/main/java/eu/europeana/corelib/definitions/model/facets/inverseLogic/MediaTypeEncoding.java

```diff
@@ -0,0 +1,35 @@
+package eu.europeana.corelib.definitions.model.facets.inverseLogic;
+
+public enum MediaTypeEncoding {
+    IMAGE(1),
+    SOUND(2),
+    VIDEO(3),
+    TEXT(4),
+    INVALID_VALUE(0);
+
+    private final int value;
+
+    MediaTypeEncoding(final int value) {
```

Comment:
Not really an issue, but an int is always final... So the final qualifier is not really required here...